docker-unified/docker-compose.yml

```diff
@@ -48,7 +48,7 @@ services:
   #   image: mcr.microsoft.com/mssql/server:2019-latest
   #   restart: always
   #   volumes:
-  #     - data:/var/opt/mssql/data
+  #     - data:/var/opt/mssql
```

Comment:
This doesnt look right to me. `/var/opt/mssql/data` is what we map on the standard deployment compose.

Reply:
It errors out on the `2019` image since they no longer use a `root` user.
https://github.com/microsoft/mssql-docker/issues/542#issuecomment-1129427855

Reply:
It still seems to be the recommended path, though?

https://learn.microsoft.com/en-us/sql/linux/sql-server-linux-docker-container-configure?view=sql-server-ver16&pivots=cs1-bash

Reply:
It's a problem when using data volume containers.  Their example on that page is the same mount as what I changed it to. https://learn.microsoft.com/en-us/sql/linux/sql-server-linux-docker-container-configure?view=sql-server-ver16&pivots=cs1-bash#use-data-volume-containers